<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NTdialog</class>
 <widget class="QDialog" name="NTdialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>252</width>
    <height>222</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Argument</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/resources/RES/ffficons/sum.png</normaloff>:/resources/RES/ffficons/sum.png</iconset>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_20">
     <property name="text">
      <string>Type</string>
     </property>
     <property name="buddy">
      <cstring>TypeComboBox</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="TypeComboBox">
     <property name="editable">
      <bool>true</bool>
     </property>
     <property name="currentIndex">
      <number>-1</number>
     </property>
     <property name="maxVisibleItems">
      <number>20</number>
     </property>
     <item>
      <property name="text">
       <string>int</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>real</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>bool</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>string</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_19">
     <property name="text">
      <string>Name</string>
     </property>
     <property name="buddy">
      <cstring>lineEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="lineEdit"/>
   </item>
   <item row="5" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_7">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>60</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="OKbutton">
       <property name="text">
        <string>OK</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="CancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Value</string>
     </property>
     <property name="buddy">
      <cstring>valueEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="valueEdit"/>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Description</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QPlainTextEdit" name="descEdit"/>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>TypeComboBox</tabstop>
  <tabstop>lineEdit</tabstop>
  <tabstop>valueEdit</tabstop>
  <tabstop>OKbutton</tabstop>
  <tabstop>CancelButton</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>CancelButton</sender>
   <signal>clicked()</signal>
   <receiver>NTdialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>218</x>
     <y>69</y>
    </hint>
    <hint type="destinationlabel">
     <x>503</x>
     <y>160</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
